cdap-tms/src/main/java/io/cdap/cdap/messaging/store/leveldb/LevelDBPartition.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright © 2021 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.cdap.cdap.messaging.store.leveldb;
+
+import org.iq80.leveldb.DB;
+
+import java.io.File;
+
+/**
+ * One partition of a MessageTable, backed by a LevelDB table.
+ */
+public class LevelDBPartition {
```

Comment:
Suggest to implement `Closeable` so that closing this class would close the underlying DB. This provides a better abstraction.